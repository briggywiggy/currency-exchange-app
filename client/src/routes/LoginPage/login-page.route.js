import React from 'react'
import UserForm from '../../components/UserForm/user-form.component';
import { startLocalLogin } from '../../actions/local-auth.action';

export const LoginPage = () => {
    return (
        <div className="login-page page">
            <div className="content-container">
                <UserForm
                title={'3DK Exam'}
                subtitle={'Login Required'}
                subtext={'Enter your e-mail and password to continue'}
                action={'login'}
                hasGoogleLogin
                submitDispatch={startLocalLogin}
                />
            </div>
        </div>
    )
}

export default LoginPage;