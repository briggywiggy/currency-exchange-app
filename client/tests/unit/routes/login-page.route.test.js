import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../../src/routes/LoginPage/login-page.route';

test('should render LoadingPage correctly', () => {
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toMatchSnapshot();
})