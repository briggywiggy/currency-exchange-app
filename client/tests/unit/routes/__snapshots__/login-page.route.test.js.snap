// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `
<div
  className="login-page page"
>
  <div
    className="content-container"
  >
    <UserForm
      action="login"
      hasGoogleLogin={true}
      submitDispatch={[Function]}
      subtext="Enter your e-mail and password to continue"
      subtitle="Login Required"
      title="3DK Exam"
    />
  </div>
</div>
`;
